content/projects/data-validation-with-python-and-regex/_index.md

```diff
@@ -1,3 +1,9 @@
 ---
-title: data-validation-with-python-and-regex
----
\ No newline at end of file
+title: data validation with Python and regex
+ready: true
+---
+
+Before you commence, firstly read through {{% contentlink "/topics/data_validation/_index.md" %}}. Upon completion, read through {{% contentlink "/topics/regular-expressions/_index.md" %}}.
+
+#### Project
+Complete the following [project]{{% contentlink "projects/tdd/string-calculator/_index.md" %}}  
```

Comment:
I think all these links will break in horrible ways. 

Please run `hugo serve` and actually look at the html that gets generated by your changes.  This is very important

Take a look at the other contentlinks in the repo as well, do any of them mention _index.md? Nope...